config.ts

```diff
@@ -16,111 +16,145 @@
 
 import * as path from 'path';
 
-// Default configuration
+/**
+ * Available configuration options.
+ */
```

Comment:
Nit. I'd prefer a one-liner for short comments like this. Like
```typescript
/** Available configuration options. */
```

Reply:
Done